src/qml/RosterPage.qml

```diff
@@ -19,15 +19,15 @@
  */
 
 import QtQuick 2.0
-import org.kde.kirigami 1.0 as Kirigami
+import org.kde.kirigami 2.0 as Kirigami
 
 Kirigami.ScrollablePage {
 	title: qsTr("Contacts")
 
 	ListView {
 		model: kaidan.rosterController.rosterModel
 
-		delegate: Kirigami.BasicListItem {  //Kirigami.SwipeListItem
+		delegate: Kirigami.AbstractListItem {  //Kirigami.SwipeListItem
```

Comment:
Yeay, right, it's ok to use it on this branch even if there is no function. I will change that soon